features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/package.json

```diff
@@ -32,6 +32,7 @@
         "draftjs-to-html": "^0.8.4",
         "file-loader": "^0.11.2",
         "html-to-draftjs": "^1.4.0",
+        "js-cookie": "^2.2.1",
```

Comment:
Why we need this library? can't we use `document.cookies` directly

Reply:
There is a method in ./Utils getCookie() method which return the relevant cookie by using document.cookies, but since it append the environment name to the given name. I'll create a separate method which returns the cookie and use that instead of above library

Reply:
removed the library and resolved the issue